stripe/src/main/java/com/stripe/android/PaymentSession.java

```diff
@@ -65,23 +65,10 @@ public PaymentSession(@NonNull Activity hostActivity) {
     }
 
     /**
-     * Complete a payment using the given provider.
-     * @param provider a {@link PaymentCompletionProvider} that connects to a server and completes
-     *                 a charge on a background thread.
+     * Notify this payment session that it is complete
      */
-    public void completePayment(@NonNull PaymentCompletionProvider provider) {
```

Comment:
I wonder if we still need to call `onPaymentSessionDataChanged()`? Probably not because this is not an async task, but worth calling out.